src/app/pages/questions/components/question/range-info-input/range-info-input.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core'
+
+import { Response } from '../../../../../shared/models/question'
+
+@Component({
+  selector: 'range-info-input',
+  templateUrl: 'range-info-input.component.html'
+})
+export class RangeInfoInputComponent {
```

Comment:
You can probably also use `export class RangeInfoInputComponent extends RangeInfoComponent`.

Reply:
Okay made the changes.